ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -1092,13 +1131,67 @@ export const DatasetPage = fp.flow(
           conceptSetId: null,
           domain: PREPACKAGED_DOMAINS[prepackaged],
         });
+        // check surveys
+        if (prepackaged === PrePackagedConceptSetEnum.SURVEY) {
+          updatedPrepackaged.add(PrePackagedConceptSetEnum.SURVEYBASICS);
+          updatedPrepackaged.add(PrePackagedConceptSetEnum.SURVEYLIFESTYLE);
+          updatedPrepackaged.add(PrePackagedConceptSetEnum.SURVEYOVERALLHEALTH);
+          updatedPrepackaged.add(
+            PrePackagedConceptSetEnum.SURVEYHEALTHCAREACCESSUTILIZATION
+          );
+          updatedPrepackaged.add(PrePackagedConceptSetEnum.SURVEYCOPE);
+          updatedPrepackaged.add(PrePackagedConceptSetEnum.SURVEYSDOH);
+          updatedPrepackaged.add(PrePackagedConceptSetEnum.SURVEYCOVIDVACCINE);
+          updatedPrepackaged.add(PrePackagedConceptSetEnum.SURVEYPFHH);
```

Comment:
Can iterate on `PREPACKAGED_SURVEY_DOMAINS` to make this a little cleaner:
```suggestion
          Object.keys(PREPACKAGED_SURVEY_DOMAINS).forEach((prepackagedSurvey) =>
            updatedPrepackaged.add(prepackagedSurvey)
          );
```

Reply:
Thanks. I tried that... maybe I had a bug. Committing you suggestion